src/main/java/simpaths/data/Parameters.java

```diff
@@ -122,6 +122,7 @@ public class Parameters {
 		"ded",					//in education dummy
 		"der",					//return to education dummy
 		"dot",					//ethnicity
+        "dot01",				//ethnicity 6 categories
```

Comment:
we have a branch that's using `dot` a lot. This will break it, but actually for the best as we'll need to ensure we update the variable to six categories :+1: